Example/ExampleSchema/ApolloSchema.graphql.swift

```diff
@@ -0,0 +1,4 @@
+// @generated
```

Comment:
Do you really need to add these generated files to git?

Reply:
Yes, we generate them only once